app/Models/User.php

```diff
@@ -18,6 +18,7 @@
 use App\Platform\Contracts\Player;
 use App\Support\Database\Eloquent\Concerns\HasFullTableName;
 use App\Support\HashId\HasHashId;
+use Carbon\Carbon;
```

Comment:
Another dumb question from me: what is the difference between `Carbon\Carbon` and `Illuminate\Support\Carbon`?

Reply:
I wondered myself some time ago 😅 
As I understood it `Illuminate\Support\Carbon` is to be used in tests where it allows to set a different time as `now()`, whereas `Carbon\Carbon` is to be used throughout the rest of the application.

Reply:
Did a little bit of research on this a few minutes ago to satisfy my curiosity. It appears Laravel extends the `Carbon\Carbon` class with `Illuminate\Support\Carbon` to add some additional Laravel-specific features to it (such as improved model date serialization support and improved localization support). However, Laravel aliases the `Carbon\Carbon` class to `Illuminate\Support\Carbon`, allowing them to be used interchangeably in a Laravel project. Mystery solved I think 😀

Reply:
oh! til - very nice, thanks :)